sdk/identity/identity/src/client/identityClient.ts

```diff
@@ -34,7 +35,10 @@ export interface TokenResponse {
    * The AccessToken to be returned from getToken.
    */
   accessToken: AccessToken;
-
+  /**
+   * The refresh token's refresh in milliseconds, UNIX epoch time.
```

Comment:
nit: it's not the refresh token. It's the time in which the access token should be refreshed.

Reply:
updated